circuit/js/src/scaffold.ts

```diff
@@ -79,12 +81,30 @@ export abstract class AxiomBaseCircuitScaffold<T> extends BaseCircuitScaffold {
     const vk = convertToBytes32(partialVk);
     const packed = encodePacked(
       ["uint8", "uint16", "uint8", "bytes32[]"],
-      [this.config.k, this.numInstances / 2, vk.length, vk],
+      [this.config.k, this.numInstances / 2, vk.length, vk as `0x${string}`[]],
     );
     const schema = keccak256(packed);
     return schema as string;
   }
 
+  prependCircuitMetadata(config: CircuitConfig, partialVk: string[]): string[] {
+    if (config.numInstance !== 1) {
+      throw new Error("`numInstance` only supported value is 1");
+    }
+    const encodedCircuitMetadata = encodeAxiomV2CircuitMetadata({
+      version: 0,
+      numInstance: [config.numInstance],
```

Comment:
this `numInstance` field refers to the actual vector of number of instances per instance column. Therefore it should be `[2 * resultsLen + 16 * max_user_subqueries]`.

Reply:
sorry - the above is incorrect `resultsLen` should be `max_user_outputs`